src/components/ContentWidgets/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ContentSingularData } from '../../_data/ContentSingularData';
+
+/**
+ * TodoList displays a list of items to be completed
+ * 
+ * Last Modified
+ * Nitesh Chetry
+ * July 3, 2019
+ */
+export const TodoList: React.FC<ContentSingularData> = ({ todoList_items }) => {
+
+    if (!todoList_items) {
+        return <></>
+    }
+    return (
+        <div>
```

Comment:
This should be a `<>` instead of `<div>`, because divs will cause a stairway. Unless it is needed to make it a block element or add styling.